olp-cpp-sdk-dataservice-read/src/repositories/PrefetchTilesRepository.cpp

```diff
@@ -126,10 +126,10 @@ RootTilesForRequest PrefetchTilesRepository::GetSlicedTiles(
       if (min_level > levels_up) {
         min_level = min_level - levels_up;
       } else {
-        // if min_level is less than steps we need to go up, go some steps down
-        auto levels_down = levels_up - min_level;
+        // if min_level is less than steps we need to go up, set min level to
+        // zero. Some quads will overlap, but we will minimize quad tree
+        // requests
         min_level = 0u;
```

Comment:
I wonder, can overlapping quad trees break our protect/release mechanism?
e.g. 2 protected quads share 1 protected tile, after releasing the tile second quad will be still protected.

Reply:
Release changed to consider all possible quads to releas